continuum/datasets/stl10.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+from torchvision import datasets as torchdata
+
+from continuum.datasets import _ContinuumDataset
+
+
+class STL10(_ContinuumDataset):
+    def __init__(
+            self, data_path: str = "", train: Union[bool, str] = True, download: bool = True):
+
+        super().__init__(data_path=data_path, train=train, download=download)
```

Comment:
for code clarity tasktype can be mentionned here (even if it is the default one)

Reply:
Done